pkg/xds/envoy/listeners/tcp_proxy_configurer_test.go

```diff
@@ -69,17 +69,15 @@ var _ = Describe("TcpProxyConfigurer", func() {
 			listenerPort:    5432,
 			statsName:       "db",
 			clusters: []envoy_common.ClusterInfo{{
-				Name:   "db{version=v1}",
+				Name:   "db",
 				Weight: 10,
 				Tags:   map[string]string{"service": "db", "version": "v1"},
 			}, {
-				Name:   "db{version=v2}",
+				Name:   "db",
```

Comment:
So where is the version tag now?

Reply:
it is picked using subset loadbalncing.

Previously when we did traffic routing we were generating separate clusters for split with this weird format. So TrafficRoute for db version=1 and db version=2, generated clusters "db{version=1}", "db{version=2}" which results in weird metrics. Now we use one cluster "db" but tcp proxy/http connection manager is configured to use metadata of endpoints

Reply:
great!